### PR TITLE
Add Drools DRL syntax highlighter

### PR DESCRIPTION
**Pull request by manstis**
Hi,

This PR includes a "Drools DRL syntax highlighter" (http://drools.org/)

I'm a newb when it comes to writing ACE Highlighters, so please guide me correcting the PR if required.

The /snippets/* files were created when I built ACE and, without them, the kitchen-sink reports HTTP404's. The ACE website's instructions on writing highlighters didn't mention snippets so IDK if they're needed in the PR or not.

With kind regards,

Mike

### CODE REVIEW

**Comment by manstis** [comment]
I work for Red Hat and hence am ATM seeking approval to the Corporate CLA...

File: lib/ace/mode/drools_highlight_rules.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            }
+        ],
+        "comment" : [
+            {
+                token : "comment", // closing comment
+                regex : ".*?\\*\\/",
+                next : "start"
+            }, {
+                token : "comment", // comment spanning whole line
+                regex : ".+"
```
Comment (nightwing) [inline lib/ace/mode/drools_highlight_rules.js]: It's better to use defaultToken like https://github.com/ajaxorg/ace/blob/master/lib/ace/mode/css_highlight_rules.js#L120-L126 does

**Comment by nightwing** [comment]
@manstis please comment here when cla is ready.

**Comment by manstis** [comment]
@nightwing Sure thing... it's currently with our legal department for review......

**Comment by manstis** [comment]
@nightwing Still chasing our legal :(

**Comment by nightwing** [comment]
Closing in favor of https://github.com/ajaxorg/ace/pull/2672/files, as we didn't get cla for this one

**Comment by manstis** [comment]
Sorry our legal department is sooo slow. I'll look to use the Drools syntax highlighter you've merged instead of this one.

**Pull request closed without merging**